src/main/java/org/certificatetransparency/ctlog/LogSignatureVerifier.java

```diff
@@ -116,58 +116,60 @@ static IssuerInformation issuerInformationFromCertificateIssuer(Certificate cert
   }
 
   /**
-   * Verifies the CT Log's signature over the SCT and certificate.
-   * Works for the following cases:
-   * * Ordinary X509 certificate sent to the log.
-   * * PreCertificate signed by an ordinary CA certificate.
-   * * PreCertificate signed by a PreCertificate Signing Cert. In this case the PreCertificate
-   *   signing certificate must be 2nd on the chain, the CA cert itself 3rd.
+   * Verifies the CT Log's signature over the SCT and certificate. Works for the following cases: *
```

Comment:
nit: Would you mind changing it so it matches the following:

```java
/**
 * Verifies the CT Log's signature over the SCT and certificate. Works for the following cases:
 * <ul>
 *   <li>Ordinary X509 certificate sent to the log.
 *   <li>PreCertificate signed by an ordinary CA certificate.
 *   <li>PreCertificate signed by a PreCertificate Signing Cert. In this case the PreCertificate
 *       signing certificate must be 2nd on the chain, the CA cert itself 3rd.
 * </ul>
 */
```

Please run the formatter again after you change it, just in case.

(Note that the `<li>s` aren't closed on purpose.)